apps/apps_fetcher.py

```diff
@@ -42,7 +43,9 @@ def get_args():
         for target, _ in apps_fetcher.target_apps.items():
             store.store(target, apps_fetcher.target_dir(target.name))
     except Exception as exc:
-        logging.error('Failed to fetch Target apps and images: {}'.format(exc))
-        exit_code = 1
+        # since preloading/fetching of App container images are not really mandatory
+        # we return successful exit code and just output an error message along with s stack trace
+        logging.error('Failed to fetch Target apps and images: {}\n{}'.format(exc, traceback.format_exc()))
```

Comment:
can we do a `logging.exception` to capture the actual stack trace. i've found myself having to make this change and then re-run ci jobs in the past to see where the actual bug is coming from.

Reply:
what's wrong with `traceback.format_exc`, as you can see in the example I provided above it prints the stack trace?

Reply:
didn't read close enough.